packages/veritone-widgets/src/widgets/UserProfile/PasswordField/index.js

```diff
@@ -15,7 +15,7 @@ const PasswordField = ({ lastUpdated, onEdit }) => {
     <div className={classes.container}>
       <RaisedTextField
         label="Password"
-        value={`Last updated ${format(new Date(lastUpdated), 'MMMM dd, yyyy')}`}
+        value={`Last updated ${format(new Date(lastUpdated), 'MMMM DD, YYYY')}`}
```

Comment:
Problem is not the format, this value is being null. Please check app repos to see why wwe are getting this value null. We also like to show different message if password is never been updated. Something like;  

```suggestion
        value={lastUpdated ? `Last updated ${format(new Date(lastUpdated), 'MMMM DD, YYYY')}` : 'Never Updated'}
```

Reply:
I logged value of `lastUpdated`, it's not null.